src/api/tokens.rs

```diff
@@ -0,0 +1,54 @@
+use actix::prelude::*;
+use actix_web::web::{Data, Json};
+use actix_web::{HttpRequest, HttpResponse, Result};
+use futures3::TryFutureExt;
+use serde::Deserialize;
+
+use crate::db::Db;
+use crate::errors::ApiError;
+use crate::tokens::{ClaimsScope, ClaimsValidator};
+
+#[derive(Deserialize)]
+pub struct TokenArgs {
+    token_ids: Vec<String>,
+}
+
+pub fn get_tokens(
```

Comment:
As Jamie pointed out, this is just as async as the other function, why have two?